docs/consensus/scheduler.md

```diff
@@ -19,6 +19,19 @@ the [consensus service API documentation].
 
 ## Validator Committee
 
+To schedule the validator committee, the committee scheduler selects among
+nodes registered with the `RoleValidator` role.
+Each node's entities must have an [escrow account balance] meeting the total
+thresholds for the nodes and runtimes that it has registered.
+If an entity's escrow account balance is too low to meet the total threshold,
+the committee scheduler does not consider that entity's nodes.
+
+From these qualifying nodes, the committee scheduler selects at most one node
+from each entity, up to a maximum validator committee size.
+Unlike how the committee scheduler schedules other committees, it schedules the
+validator committee by choosing nodes from the entities that have the highest
```

Comment:
Maybe mention (and link to) parameters which specify K.

Reply:
ok